rdf4h.cabal

```diff
@@ -124,6 +124,8 @@ test-suite test-rdf4h
                , filepath
                , directory
                , safe
+               , temporary
```

Comment:
@tmciver  It seems a slight shame that this introduces an additional dependency, which itself brings in its own dependencies that rdf4h doesn't rely on (random, transformers, unix), though these may be transitive dependencies from rdf4h's existing dependencies.

That said, it's a dependency of the rdf4h test-suite, not of the rdf4h library. So for most users just depending on the rdf4h library won't pull those additional dependencies in via `temporary`.

Reply:
Yes, I figured too that since it's in the testsuite, that it wasn't a big deal.

Reply:
Also, I wanted to ask about your thoughts on exposing functions from modules.  In PR #95 I had exposed `findMapping` and `writeUNodeUri` which were not previously exported.  It's a shame, IMO, to have to do this; I wish Haskell had a way of only exposing functions to the test package, or something like that.

There are a few options.  First, we could not expose such functions and just test them through the functions that are exported.  Another would be to use often-used technique of having a set of `Internal` modules where everything is exposed and we just rely on convention to keep users from depending on the things exposed there.  Finally, we could just expose more such functions and consider it part of the API.  Thoughts?

Reply:
@tmciver Iet's move this discussion over to https://github.com/robstewart57/rdf4h/pull/95